lib/persisted-model.js

```diff
@@ -639,11 +640,14 @@ module.exports = function(registry) {
     setRemoting(PersistedModel, 'create', {
       description: 'Create a new instance of the model and persist it into the data source.',
       accessType: 'WRITE',
-      accepts: {
-        arg: 'data', type: 'object', model: typeName,
-        description: 'Model instance data',
-        http: { source: 'body' },
-      },
+      accepts: [
+        {
+          arg: 'data', type: 'object', model: typeName,
+          description: 'Model instance data',
+          http: { source: 'body' },
+        },
+        { arg: 'options', type: 'object', http: optionsFromContext },
```

Comment:
Will this be handled correctly by the swagger generation side of things? Is there something we can add here to hint that this argument is not an API parameter?